packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -133,6 +134,24 @@ export class OutgoingPayment
 
   public tenantId!: string
 
+  public dataToTransmit?: string
+  public getDataToTransmit(key?: string): string | null {
+    if (!this.dataToTransmit) return null
+    if (!key) return this.dataToTransmit
+    const { tag, cipherText, iv } = JSON.parse(this.dataToTransmit)
```

Comment:
Key rotation might be a bit tricky here, since we could end up encrypting with one key, change the key, try to decrypt with the old one and fail. We can focus on this on a follow-up PR.
(maybe we allow to configure a list of keys instead?)

Reply:
Captured in [RAF-1207](https://linear.app/interledger/issue/RAF-1207/handle-key-rotation-for-existing-data).